### DISCUSSION TITLE
4/24 - Vue組件基礎和溝通

### DISCUSSION THREAD

**Discussion by yhosutun2490** [Tech-Book-Community/Vue3#8] [General]
簡報:
https://hackmd.io/@tPmnebB_ScehLt-q3uTXnw/BkA-bgq-R

**Comment by janlin002** [comment]
[props 示例](https://play.vuejs.org/#eNqVVM9r2zAU/leELmmhsfsLBp4TaEcP7WEt3W7TDq4tt2oVSVhy1hICg1HaHQY9bYwVdtpxp0EZG/tvmrR/xp5kx3FKEgiEWHrf06dPet9TD28p5XVzigMc6jhjyiBNTa7aRLCOkplBPZTRFPVRmskOakBqo4K2pTwr455vJ5ap8ZwIImIptEFJZKKWXb/UIwKhI0jRwRs7dHOEWBKgtZViLKIODVDjMEojyl1uo0Si3JzIrMJGYWC3C9ZX1zebq/BbA2UW6Tu8tsP65A57NE2n81tkFvuzWewbk+xbIrmYzm6RGexrmyN2+/eWiP4yEaFfVARqARNDO4rDKpghFKr2w/Xd4PLD/d+7x19Xw89fQ9/WDJCco24zlVmLYKsCMeHK4LnLJxgFZ/SixDyWEOxWwTrOylGv5+R79jyo7wSNg8VhnkbtaarU0C+pQj/nBSfIHfy8mSrXeWghwShwSHNXpLKME+wDW+jX7givYKPBhSk79k61FGBwVzOCY9lRjNNsXxkGLiU4GFWT4Ihz+W7PxUyW07JYsOaExmdT4qf63MYIPsioplmXElxhJsqOqSngnVcv6TmMK7Ajk5xD9hzwkGrJc6uxSNvORQKya3lO7a7rRSaOX+udc0OFHh3KCh0blmDozRdzjj6Wu+Ftllbswy2O+nreC5HQlAl6kEmlJ14KeAqKh0A5qFVPrD0JtpKVEnOhoDP2j05pbEbNQtMo5yZAS8uo1UY9ZzQrD5pkTpuECeuimEdag03iE8aTZmGWwnmAtofffg9uPgUoZELlxu0NucaVA1wJhaAcAk5+YUir1bWG9VzoW5KK7f7f7eP7y4XZip6awje4+vPw8Xr45cfw9vvCrLYn65zjb71L+v8Bg90FrQ==)

[emit 示例](https://play.vuejs.org/#eNp9VMtuEzEU/RXjRSeRQmYBqzCNoFBEkXgIkNh4M525E9x67JHtSVtF8wEIJPgAFlRixZIFi/I7VST+gmt7ZpKQNjv7nuPjcx/2gj6qqvG8Bjqhick0rywxYOtqyiQvK6UtWRANBWlIoVVJIqRGPfTi4qC2VskWG8ddwClGD5hkMlPSWFKaGdl3OoPoGQihyHulRX4nGvaElMsnqU1bFqOMIpbEwRKawY2FshKpBdwRslh0R5rG7ZPey0NTH5fc7jM6yBEe7k/X1V2I0WnSW0W1JF6TpiNqDZoq+Gx8YpTEwizcBYxmqqy4AP2qshxNMzohHnFYikmdPfcxq2sYdfHsA2SnN8RPzLmLMfpagwE9B0Z7zKZ6BjbAh29fwjmue7BUeS2QvQN8A0aJ2nkMtINa5mh7jefdHvkecjl7Zw7PLUjTJeWMOmbj+YxiLx/vSH1l9974vj/HZINVXJ+FXdOVQ8ElHGLLzOj/ScMRCvNRKF22s+EvBmypmJAo8h6r1JgzpXMXwMv7qQIUdT1f3RBOxzFZ/rpcfvu4/H759+dnF8oEz04nRNZCoGRH8jAJA0Wurz5d//ntoBCYkMEiGBn1DkgzJDhwbXF4QQaeQPb2esqwQwmmY2st+4Jj6QgIAyuCS0MJGJ+lWg6iIzlPBc87PzAHaVH2Qqg0vKUN1SJFqVa2awtymCxqmbk2tjpPsbTB5SqN1qOr3yAKtGiEvdrOFkX947z9pSY5n/sFLk2VyqkXwQNu3ca5rGpL5ndxgEHgy3XtHnseo3FQiXuZLcHOzobmLaIdd1s3OW7/Dz8KyF8rz8rPaENliF/J8svX66sfSRyO+wIE4fVPpfkHBxzcpA==)

[slot 示例](https://play.vuejs.org/#eNp9U9tu00AQ/ZXpVqggxQnNhUswldqqSOUBEPDoF8cem03Wu6u9tImqfANfwBsfwffwA/wCs+vGOBHtm+fMmZkzZ9Z37Fzr4Y1HNmepLQzXDiw6r88yyRutjIN3uSw2F945JaEyqoGT4aiHheKTjnx+i1Y1eF30yD3snpyO2lE0hAKHjRa5Q4oA0l7rCABcCl6soEGKMpf2Zz9UkVqdS7BuI/BtxgollJkXm1xm7GzX7AhIBLF2FX3ho7bvwaR01AllA3bgQLBvf49F61ghcmtJRBX4ycIFDSFPm1ihXBgVlmrZ+1OiOXEJ+hx2DeAu1LdLwXFVVW9CvMiLVW2Ul+UcBJeYm6Q2eclRuqeT01mJ9QCOp+Ny8noM49kTCl5MX2JVPWurlSmR2kklMQI6L0su6znM9BpOn+t1RJvc1FxGsFeWhDnezuHVPVx4Y4M4rbh0aAjbxpvHVci7gwcRvDtKEvCWJgIdDhu15LD01kGlDJQUg/ZGK4sWkiS41Jn0+8fPP7++79/GWepc8Xq4tEpS92hYeAeN5gLNR+24kjZj89bKkMuFULfvI+aMx8EOL75hsfoPvrTrgGXsk0GL5gYz1uUcuYSuTV99+YBr+u6SjSq9IPYjyc/kjvBBY0u7oKOS7B4vqr2OfxxZ9tVerR1Ku1sqCA3MbeRnjCy+fGT1f3Inw2mso3ux7V9TJ2PO)
